plugin/mutate/doc/design/options.md

```diff
@@ -0,0 +1,220 @@
+
+
+- --id : Mutate the source code as mutant ID
+
+
+## Report
+*Report-mode for the plugin. Is used after mutation testing has been conducted in order to generate a result-report. Can also be used to generate specific result that helps a user improve test cases among other.*
+
+
+- --compile-db : Retrieve compilation parameters from a specific compilation-database. This can be used if the projects contains specific compilation-databases for ex. compilation targets or environments.
+
+
+- --diff-from-stdin : Report alive mutants in the areas indicated as changed in the diff.
+
+
+- --level : Report level of the mutation data.
+    - *summary* : Create a report that is a summary of the result.
+    - *alive* : Create a report that only lists the alive mutants.
+    - *all* : Create a report that contains all the mutants.
+
+
+- --logdir : Directory to write log files to (default: .).
+
+
+- --mutant : Kind of mutation to report.
+    - *any* : Any mutant generated.
+    - *ror* : Relational Operator Replacement.
+    - *rorp* : Relational Operator Replacement (Pointer).
+    - *lcr* : Logical Connector Replacement.
+    - *lcrb* : Logical Connector Replacement (Bit-wise).
+    - *aor* : Arithmetical Operator Replacement.
+    - *uoi* : Unary Operator Insertion.
+    - *abs* : Absolute Value Insertion.
+    - *sdl* : Statement Deletion.
+    - *cor* : Conditional Operator Replacement.
+    - *dcc* : Decision/Condition Coverage.
+    - *dcr* : Decision/Condition Replacement.
+
+
+- --restrict : Restrict analysis to files in this directory tree (default: .). This option can be used to make sure that mutations are not generated for specific files outside a specific directory tree. This option together with a generated compilation-database specified with *--compile-db* lets *Mutate* iterate over every file in the project, and compares their paths to the *restricted area*.
+
+
+- --section : Sections to include in the report.
+    - *alive* : TODO
+    - *killed* : TODO
+    - *all_mut* : TODO
+    - *summary* : TODO
+    - *mut_stat* : TODO
+    - *tc_killed* : TODO
+    - *tc_stat* : TODO
+    - *tc_map* : TODO
+    - *tc_suggestion* : TODO
+    - *tc_killed_no_mutants* : Provide a list of tests that killed no mutant.
+    - *tc_full_overlap* : Provide a list of tests that killed the exact same mutants (candidates for redundant test cases).
+    - *tc_full_overlap_with_mutation_id* : Provide a list of tests that killed the exact same mutants (candidates for redundant test cases), but include the id of the mutants.
+    - *tc_groups* : TODO
+    - *tc_min_set* : Provide the minimal set of test cases needed in order to achieve the mutation score.
+    - *tc_similarity* : Provide a list of tests and to what degree they are similar in terms of mutants the kill.
+    - *tc_groups_similarity* : TODO
```

Comment:
Compare the similarity between test groups. This is a "group" view compared to tc_similarity